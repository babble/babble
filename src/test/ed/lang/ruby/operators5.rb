
bar = false;
puts bar
bar ||= true;
puts bar

bar = true
foo = false;
puts foo
foo ||= bar;
puts foo

def blah( a , b )

  if ( a && b )
    puts "and"
  end

  if ( a || b )
    puts "or"
  end
end

blah( true , false );
blah( false , true );
blah( false , false );
blah( true , true );
