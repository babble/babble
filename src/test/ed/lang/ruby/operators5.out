false
true
false
true
or
or
and
or
yay
----
false
true
true
